### PR TITLE
Javascript/javascript1/week1

### PR DESCRIPTION
**Pull request by ajgcph2019**
Freecodecamp exercises also done .
Username : ajgcph2019

### CODE REVIEW

File: javascript/javascript1/week1/dog_age.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const dogYearOfBirth = 1957;
+const dogYearFuture = 2027;
+console.log(dogYearFuture - dogYearOfBirth);
+const dogYear = 60;
+const dog_age= dogYearFuture - dogYearOfBirth;
```
Comment (braedongough) [inline javascript/javascript1/week1/dog_age.js]: Remember to always use [camel case](https://en.wikipedia.org/wiki/Camel_case) in Javascript.

File: javascript/javascript1/week1/dog_age.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+if (dog_age <=dogYear) { 
+ shouldShowResultInDogYears = true;
+console.log("Your dog will be " + dogYear + " dog years old in " + dogYearFuture);
+ }
+else{
+shouldShowResultInDogYears = false;
```
Comment (braedongough) [inline javascript/javascript1/week1/dog_age.js]: Remember to always declare your variables. If you plan on re-assigning this one you could declare 
```js
let shouldShowResultInDogYears = true
```

at the top of file and re-assign it to false when needed.

File: javascript/javascript1/week1/house.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+let firstHouseWidth = 8;
```
Comment (braedongough) [inline javascript/javascript1/week1/house.js]: You should make sure to use `const` if you don't plan on re-assigning a variable.

File: javascript/javascript1/week1/house.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+let firstHouseHeight = 10;
+let gardenSizeInM2 = 100;
+let volumeInMeters;
+let sellingPrice_Peter = 2500000;
+let sellingPrice_Julia = 1000000;
+let secondHouseWidth = 5;
+let secondHouseDepth = 11;
+let secondHouseHeight = 8;
+let secondGardenSizeInM2 = 70;
+volumeInMeters = firstHouseDepth * firstHouseWidth * firstHouseHeight ;
```
Comment (braedongough) [inline javascript/javascript1/week1/house.js]: I would also recommend that you add space between some lines to make your code easier to read. Group variables at the top like you did is great, however it's easier for others to review when you provide space between different types of statements such as re-assigning a variable or creating a conditional.

**Comment by ajgcph2019** [comment]
Thank you Braedon for the feedback. I have made the changes now.

**Pull request merged**